changes/pr123.yaml

```diff
@@ -0,0 +1,2 @@
+feature:
+  - "Add experimental Helm chart for deploying in K8s - [#123](https://github.com/PrefectHQ/server/pull/123)"
```

Comment:
Would you mind including a `contributor` section here that lists @shaunc , @yolibernal and @joelluijmes as contributors?  Definitely want to recognize their effort in kicking this off and QA'ing

Reply:
Great idea! Will do

Reply:
> Minor request for changelog update; also noticed `helm/prefect-server/charts/postgresql-9.3.2.tgz` was included - was that intentional?

I tried removing it and then `helm install ...` failed, I'll look into it a bit more.

Reply:
Fixed :)

Reply:
Good catch, I didn't notice that. Yeah, we shouldn't vendor this, dependencies should be managed externally. Note that just deleting the file won't be sufficient to reduce the repo size (not sure if we care about this), since the commit that adds it will still be present.

Reply:
I did a rebase and dropped it from the commit that added it -- my git-foo is only medium strength though so perhaps worth someone else checking it's gone.

Reply:
It's still in the history, but it's also only 44k in size so I think it's OK